Fixed wet/dry override & code clean-up

ronvandegraaf opened the pull request:
- Fixed wet/dry function
- Changed labels open/close to wet/dry
- Small code clean-up, removed tabs
- Tested device handler

gn0st1c commented:
this is great. i, too, don't like trailing spaces, not aligned stuff and so on.
this code now looks pretty much the same as mine :)

except for the tabs vs spaces, i'm pro tabs.
https://www.youtube.com/watch?v=SsoOG6ZeyUI

ronvandegraaf commented:
Added extra commit to this pull request: Changed the colors for dry (=blue) and wet (=orange), like the door/window sensors. To give more attention in your 'Things' list.

veeceeoh commented on devicetypes/bspranger/xiaomi-aqara-leak-sensor.src/xiaomi-aqara-leak-sensor.groovy:
```diff
@@ -47,63 +47,63 @@ metadata {
         }
-        standardTile("resetClosed", "device.resetClosed", inactiveLabel: false, decoration: "flat", width: 2, height: 1) {
-            state "default", action:"resetClosed", label: "Override Close", icon:"st.contact.contact.closed"
+        standardTile("resetWet", "device.resetCWet", inactiveLabel: false, decoration: "flat", width: 2, height: 1) {
```
"device.resetCWet" should be "device.resetWet"

veeceeoh commented on devicetypes/bspranger/xiaomi-aqara-leak-sensor.src/xiaomi-aqara-leak-sensor.groovy:
```diff
@@ -47,63 +47,63 @@ metadata {
         attribute "lastOpened", "String"
```
This attribute should be changed to "lastWet"
Sorry - I also missed this in yesterday's push!

veeceeoh commented on devicetypes/bspranger/xiaomi-aqara-leak-sensor.src/xiaomi-aqara-leak-sensor.groovy:
```diff
@@ -47,63 +47,63 @@ metadata {
         attribute "lastOpened", "String"
         attribute "lastOpenedDate", "Date"
```
This attribute should be changed to "lastWetDate"

veeceeoh commented:
Apologies! I see that I missed fixing some of the open / close references in my push yesterday. Looks like you caught nearly all of it now - there's just a couple of attributes defined in the metadata section that also need to be changed.

ronvandegraaf commented:
Changed it in my last commit. We maken some great progress on these device handlers now.

veeceeoh commented:
ranvandegraff - do you have problems with the appearance of the Battery Change tile? It looks incorrect in the ST App on my iPhone. I will post screenshots on Brian's ST forum thread.

ronvandegraaf commented:
Yes, same problem for me (iPhone Plus).

On Wed, 17 Jan 2018 at 21:17, Keith G <notifications@github.com> wrote:

> ranvandegraff - do you have problems with the appearance of the Battery
> Change tile? It looks incorrect in the ST App on my iPhone. I will post
> screenshots on Brian's ST forum thread.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bspranger/Xiaomi/pull/23#issuecomment-358431148>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAGTlkUNET2quJBEMx4VBq9euLjGbal3ks5tLlVpgaJpZM4Rg5rP>
> .
>
